emva1288/tests/test_camera_pytest.py

```diff
@@ -0,0 +1,126 @@
+import pytest
```

Comment:
Why not add tests folder outside of emva1288 folder?

Reply:
It could be a good to separate them. I simply created the Tests folder to be next to the  Unittests folder for comparison

Reply:
I have refactored tests to be in the root of the project now